docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -83,6 +83,37 @@ ALTER TABLE table_identifier ADD COLUMNS ( col_spec [ , col_spec ... ] )
 </dl>
 
 
+### ADD PARTITION
+
+`ALTER TABLE ADD PARTITION` adds a new partition to an existing, partitioned table.
+
+#### Syntax
+
+```sql
+ALTER TABLE <table_identifier> ADD [IF NOT EXISTS]
+  PARTITION <partition_spec> [LOCATION 'location']
+  [ PARTITION <partition_spec> [LOCATION 'location'] ...]
+;
+
+<table_identifier> ::=
+  [ database_name. ]table_name
+
+<partition_spec> ::=
```

Comment:
I know we currently have parameters described in a separate section, but I believe having the syntax all described at once is clearer. This also follows the style of [Microsoft's SQL Server docs](https://docs.microsoft.com/en-us/sql/t-sql/statements/create-table-transact-sql), which I believe are a great model to follow. 

But if we really want to stick to the separate "Parameters" section with HTML-based lists, I can switch to that. But I will likely propose in a separate PR that we eliminate raw HTML from these SQL docs unless they are absolutely necessary.